src/ocean/util/app/ext/ReopenableFilesExt.d

```diff
@@ -70,14 +93,27 @@ public class ReopenableFilesExt : IApplicationExtension, ISignalExtExtension
 
     ***************************************************************************/
 
-    public this ( SignalExt signal_ext = null, int reopen_signal = SIGHUP )
+    public this ( SignalExt signal_ext = null,
```

Comment:
Oh wait... this is a breaking change, or?

Reply:
Puh, yeah, I've missed that - I originally targeted v4.x.x. Just having new constructor with suffice, though. Will update.

Reply:
Ah, I guess it's registered twice because of this... Maybe add an issue and a comment to clean this up in a following major?